src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/topology/utils/LoggerContextInitializer.java

```diff
@@ -51,6 +53,28 @@
 public class LoggerContextInitializer {
     private static final Logger LOGGER = LoggerFactory.getLogger(LoggerContextInitializer.class);
 
+    private String stormId;
+
+    public LoggerContextInitializer() {
+        this.stormId = System.getProperties().get("storm.id") != null
+                ? getCleanTopoName(System.getProperties().get("storm.id").toString())
+                : "";
+    }
+
+    static String getCleanTopoName(String stormId) {
+        if (stormId == null || stormId.isEmpty()) {
+            return stormId;
```

Comment:
Too many null checks (here and on line 59), collect them in one place.

Reply:
cleaned